components/partials/portfolio/bridge/token-selector/select.vue

```diff
@@ -196,6 +207,10 @@ export default Vue.extend({
   },
 
   methods: {
+    onMaxInput(percent: Number) {
+      this.$emit('input:max', percent)
```

Comment:
as described in https://github.com/InjectiveLabs/injective-dex/pull/676/files#r845947143

lets do this instead `this.$emit('input:max')`